tracee-ebpf/tracee/argprinters.go

```diff
@@ -176,7 +176,72 @@ func (t *Tracee) prepareArgs(ctx *context, args map[string]interface{}) error {
 		if cmd, isInt32 := args["cmd"].(int32); isInt32 {
 			args["cmd"] = helpers.ParseBPFCmd(cmd)
 		}
+	case SecurityKernelReadFileEventID:
+		if readFileId, isUint32 := args["type"].(uint32); isUint32 {
+			typeIdStr, err := ParseKernelReadFileId(int32(readFileId))
+			if err != nil {
+				return err
+			}
```

Comment:
We currently don't return an error if an argument parsing failed (also see other argument parsers in this function) - let's just return an empty string in this case

Reply:
i want to use the enum value if we couldn't translate to str (so this data won't be totally lost). 
so i changed it to be updated only if `err == nil`